task-graphs-v2.yaml

```diff
@@ -366,6 +366,152 @@ definitions:
     description: >
       Placeholder to reserve the `UDFArrayDetails` name before merging these
       messages into `openapi-v1.yaml`.
-
-# Empty paths object to make this a valid OpenAPI document before merging.
-paths: {}
+  UDFSharing:
+    description: Placeholder to reserve `UDFSharing`.
+  Error:
+    description: Placeholder to reserve `Error`.
+
+paths:
+  /taskgraphs/{namespace}/registered/{name}:
```

Comment:
- We should probably consider these read-only—maybe the server writes them in responses but never reads them.
- Access control works the same as in UDFs.
- The naming is based on the endpoint for UDFs, which has the same parameters, I presume since we handle permissions a bit differently.